MAUI/Introduction/Overview.md

```diff
@@ -26,6 +26,6 @@ Similar to the Knowledge Base, the Forum section also contains responses to ques
 
 ## Support and feedback
 
-If you are unable to find the information that you are looking for in the self-help resources mentioned above, please contact us by creating a [support ticket](https://www.syncfusion.com/support/directtrac/incidents).
+If you are unable to find the information that you are looking for in the self-help resources mentioned above, please contact us by creating a [support ticket](https://www.syncfusion.com/Account/Login?ReturnUrl=%2fsupport%2fdirecttrac%2fincidents).
```

Comment:
https://support.syncfusion.com/create - This is our latest URL. Please make a note of it. Change the url.

Reply:
Updated the correct URL